flink-runtime/src/main/java/org/apache/flink/runtime/io/network/NetworkEnvironment.java

```diff
@@ -287,4 +311,8 @@ public boolean isShutdown() {
 			return isShutdown;
 		}
 	}
+
+	public static NetworkEnvironment build(NetworkEnvironmentConfiguration configuration) {
```

Comment:
If it is used only for tests, I would suggest to move to test packages. It could be `NetworkEnvironmentConfigurationBuilder.buildNetworkEnvironment()` or maybe we could just rename  `NetworkEnvironmentConfigurationBuilder` to `NetworkEnvironmentBuilder` if we do not need builder only for `NetworkEnvironmentConfiguration`.

Reply:
Yes, I think it is better to rename `NetworkEnvironmentConfigurationBuilder` to `NetworkEnvironmentBuilder` and migrate this build there.